notebooks/DrizzlePac/sky_matching/sky_matching.ipynb

```diff
@@ -4,14 +4,27 @@
    "cell_type": "markdown",
```

Comment:
<p>Same here. I don't see a plot being displayed, but it's up to you to decide.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>